src/less/components/sheet.less

```diff
@@ -49,14 +49,22 @@
 
 body.sheet-open {
     overflow: hidden;
-    padding-right: 17px; // Width of the scrollbar (don't change this) [EH]
+    // padding-right: 17px; // Width of the scrollbar (don't change this) [EH]
```

Comment:
You can remove these lines instead of commenting them out if they are no longer required.